docs/user/guides/build/webhooks.rst

```diff
@@ -1,46 +1,28 @@
-How to setup build notifications and webhooks
-=============================================
+How to setup build status webhooks
+==================================
```

Comment:
I guess we should be systematic about calling the feature "build status webhooks" since that's how it is right now.

If we're gonna do anything more drastic, I think it should include the Dashboard terminology, so I would just save that for later..